config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -185,6 +185,11 @@ config CHIP_QSPI_NOR
     bool
     default y
 
+# Enable extended discovery
+config CHIP_EXTENDED_DISCOVERY
+    bool
+    default y
```

Comment:
Should this default to "n", with people who want the feature then being able to enable it?  This feature is meant for a fairly specific set of use cases, as far as I can tell, and most devices will probably not want it...

Reply:
Yes I agree, however we want to have it enabled for the nRF examples. So with the current implementation it is disabled by default for Zephyr and nRF connect platform according to this definition: https://github.com/project-chip/connectedhomeip/pull/18367/files#diff-bc10c06849de8756cfdadd674dbc3b8775bc75a337a054f962365defa05794daR247. What the Kconfig.defaults does is overriding defaults for nRF Connect examples, so platform has it disabled by default and we enable it only for our examples. In case somebody would like to create own example based on Zephyr/nRF connect platform he/she will have it disabled.